kubejs/startup_scripts/Material_Registry.js

```diff
@@ -1,16 +1,17 @@
 // Elements
 GTCEuStartupEvents.registry('gtceu:element', event => {
+    event.create('infinity', 69, 420, -1, null, '∞', false); //feel free to add a better proton/neutron count
```

Comment:
```suggestion
```
dont make an **element** since nobody "inherits" infinity ingot thru `.components()`

Reply:
Why does it matter tho? Nobody inherits omnium either, and it got an element anyways.

Reply:
Marked as resolved because all single-element materials have their element, even if there isn't any alloy using them (souce: modpack made by GTM dev)